assets/app/scripts/controllers/project.js

```diff
@@ -31,9 +32,9 @@ angular.module('openshiftConsole')
         function(e) {
           $scope.projectPromise.reject(e);
           if (e.status == 403 || e.status == 404) {
```

Comment:
Tried to test this a few different ways but was unsuccessful.  Did not want to change it to === if **working** depends on coercion.